components/extractors/ip-addresses/kbc-public-ip.json

```diff
@@ -1,6 +1,6 @@
 {
```

Comment:
nešlo by sem dát
```
redirect_to:
  - components/kbc-public-ip.json
```
?

Reply:
Vyzkouším

Reply:
Nevalí mi to, asi zatím neřeším a nechám takhle.

Reply:
ok